_content/monographs/along-the-ancient-silk-routes_hartel-herbert-yaldiz-marianne.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Along the Ancient Silk Routes: Central Asian Art from the West Berlin State Museums"
+authors:
+  - "Herbert Härtel"
+  - "Marianne Yaldiz"
+external_url: "https://cdn.sanity.io/files/cctd4ker/production/93ca9a62747483bb5f6240e41122b53e823ac5ff.pdf"
```

Comment:
Let's add the met page as a source url: https://www.metmuseum.org/met-publications/along-the-ancient-silk-routes-central-asian-art-from-the-west-berlin-state-museums